### ISSUE TITLE
Fading enchantment high cpu usage

### ISSUE THREAD

**Issue by someaddons** [Darkhax-Minecraft/Cursed#26]
![image](https://user-images.githubusercontent.com/38401808/122393974-c0574480-cf75-11eb-9f77-cb269f0ec61c.png)

The fading enchantment uses relatively much tick time for what its supposed to do, due to looping all entities each tick. I think you could just do it once every 20 ticks or so and increase the lifetime reduction accordingly

**Comment by Darkhax** [comment]
I don't think this CPU usage is bad, but it certainly could be better. I will probably use a Mixin for this effect to make the performance impact near non-existent.
